01-Login/README.md

```diff
@@ -68,4 +75,1059 @@ If you have found a bug or if you have a feature request, please report them at
 
 ## License
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE.txt) file for more info.
+This project is licensed under the MIT license. See the [LICENSE](../LICENSE) file for more info.
+
+# Tutorial
+This tutorial will help you get up and running the React app using the new SDK.
+
+## Setting Up the Application
+
+### Installing initial dependencies
+After create a new React app using `create-react-app` we still need to install some dependencies that doesn't come with the boilerplate. These dependencies will help us with things like routing, styling and font icons. But, before that we need to move the current `dependencies` to `devDependencies` this is because we will be using a server to run the built app.
```

Comment:
I would actually remove the need for the reader to move everything into `devDependencies`. I know we do that for the sample but I'm not sure it's a detail that the reader has to worry about unless they're going to be running this app in production.

Also let's drop the allusions to the styling and fonts, as that's a detail specific to the sample app only (see a later comment about the dependencies).

```suggestion
After create a new React app using `create-react-app` we still need to install `react-router`, which doesn't come as standard with the boilerplate project.
```